package/batocera/emulators/retroarch/libretro/libretro-mrboom/libretro-mrboom.mk

```diff
@@ -0,0 +1,20 @@
+################################################################################
+#
+# MRBOOM
+#
+################################################################################
+# Version.: Commits on Feb 28, 2019
```

Comment:
you can remove this comment when using a specific version

Reply:
I think my linux build server(Ryzen 5 1600) is suffering from segfault's. I can't boot it up until I get home. I'll post my changes to see what needs to be adjusted, but I did most of what @nadenislamarre  suggested